README.md

```diff
@@ -213,6 +214,45 @@ E.g. only output the `inverse` variant if the brand supports it:
 }
 ```
 
+### oBrandCustomize
+
+`oBrandCustomize` enables projects to modify brand variables which were configured within a component using `oBrandDefine`. But each component must provide an adapter (wrap the brand mixin in their own mixin), as `o-brand` must not be used directly outside Origami components.
```

Comment:
```suggestion
`oBrandCustomize` allows existing brand variables to to be modified. If a brand variable was defined with `oBrandDefine`, it can be changed with `oBrandCustomize`. This customisation is component-specific, so every component must wrap `oBrandCustomize` within a mixin of its own, as `o-brand` must not be used directly outside Origami components.
```